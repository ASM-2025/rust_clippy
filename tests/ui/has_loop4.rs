#![allow(unused)]
#![warn(clippy::has_loop)]

fn main() {
    for i in 1..4 {
        println!("{}", i);
    }
}
