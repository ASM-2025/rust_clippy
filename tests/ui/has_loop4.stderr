error: Loop detected
  --> $DIR/has_loop4.rs:5:5
   |
LL | /     for i in 1..4 {
LL | |         println!("{}", i);
LL | |     }
   | |_____^
   |
   = note: `-D clippy::has-loop` implied by `-D warnings`

error: aborting due to previous error

